### PR TITLE
kerneltest: add stderr log file

### PR DESCRIPTION
**Pull request by NDStrahilevitz**
## Initial Checklist

- [ ] There is an issue describing the need for this PR.
- [ ] Git log contains summary of the change.
- [ ] Git log contains motivation and context of the change.
- [ ] If part of an EPIC, PR git log contains EPIC number.
- [ ] If part of an EPIC, PR was added to EPIC description.

## Description (git log)

Log file is printed at the end of the test. It is used to help debug
regressions and potentially discover other problems.

Fixes: #issue_number

## Type of change

- [ ] Bug fix (non-breaking change fixing an issue, preferable).
- [ ] Quick fix (minor non-breaking change requiring no issue, use with care)
- [ ] Code refactor (code improvement and/or code removal)
- [x] New feature (non-breaking change adding functionality).
- [ ] Breaking change (cause existing functionality not to work as expected).

## How Has This Been Tested?

Tests being included in this PR:

- [ ] Test File A
- [ ] Test File B

Reproduce the test by running:

- command 01
- command 02

## Final Checklist:

Pick "Bug Fix" or "Feature", delete the other and mark appropriate checks.

- [ ] I have made corresponding changes to the documentation.
- [ ] My code follows the style guidelines (C and Go) of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented all functions/methods created explaining what they do.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix, or feature, is effective.
- [ ] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published before.

## Git Log Checklist:

My commits logs have:

- [x] Subject starts with "subsystem|file: description".
- [x] Do not end the subject line with a period.
- [x] Limit the subject line to 50 characters.
- [x] Separate subject from body with a blank line.
- [x] Use the imperative mood in the subject line.
- [x] Wrap the body at 72 characters.
- [x] Use the body to explain what and why instead of how.

### CODE REVIEW

File: tests/kerneltest.sh [hunk 1]
```diff
@@ -87,6 +92,7 @@ for TEST in $TESTS; do
       -o format:gob \
       -o option:parse-arguments \
       -o option:detect-syscall \
+      -o err-file:${ERROR_FILE} \
```
Comment (rafaeldtinoco) [inline tests/kerneltest.sh:95]: We should place the stderr file in /tmp together with "/tmp/build-$$" file. OR make sure all temp files are in the same place (and with same naming convention, like using a pid attached to temp file name).

At the end of the script we should clean both files (I forgot to remove /tmp/build-$$ as well). I would use /tmp/tracee for tracee temporary files only.

Comment (NDStrahilevitz) [reply tests/kerneltest.sh:95]: Error file is under /tmp/tracee, and there is an `rm ${ERROR_FILE}` at the end.

**Comment by NDStrahilevitz** [comment]
Not sure if this isn't working or there are simply no errors, need to investigate further.

**Comment by rafaeldtinoco** [comment]
I have implemented this at https://github.com/aquasecurity/tracee/pull/2029

**Pull request closed without merging**